SPEC.md

```diff
@@ -266,6 +266,26 @@ line of the file.
 
 Offsets, however are 0-based. A Position of `{ "offset": 4 }` represents the _fifth_ character in the file. Importantly, the `offset` is from the beginning of the file, not the beginning of a line. Newline characters (and all characters) count when computing an offset.
 
+### Other locations
+
+Other locations is an optional array of location objects
```

Comment:
1. Missing a period here.
2. `location` should be capitalized & linked to the `Locations` section that describes the Location schema.